### PR TITLE
Traitor Panel now shows the names of blood brothers' brothers.

### PR DESCRIPTION
**Pull request by harmonyn**
## About The Pull Request

Exactly what it says in the title, TP now lists blood brothers' names. 
## Why It's Good For The Game

Because it helps admins doing investigations and looking in TP. Feel free to suggest other antag shit that should be in TP and isn't.

## Changelog
:cl:
admin: TP now shows the names of blood brothers' brothers.
/:cl:

### CODE REVIEW

**Comment by Coconutwarrior97** [comment]
thank god, this is incredibly based

File: code/modules/antagonists/brother/brother.dm [hunk 1]
```diff
@@ -31,22 +31,30 @@
 	owner.special_role = null
 	return ..()
 
-/datum/antagonist/brother/proc/give_meeting_area()
-	if(!owner.current || !team || !team.meeting_area)
-		return
-	to_chat(owner.current, "<B>Your designated meeting area:</B> [team.meeting_area]")
-	antag_memory += "<b>Meeting Area</b>: [team.meeting_area]<br>"
+/datum/antagonist/brother/antag_panel_data()
+	var/brother_text = get_brother_names()
```
Comment (AnturK) [inline code/modules/antagonists/brother/brother.dm]: Don't need this var here.

**Review by AnturK — DISMISSED** [review]
Looks good, just some code golf suggestions

File: code/modules/antagonists/brother/brother.dm (hunk 1 above)
Comment (bgobandit) [reply code/modules/antagonists/brother/brother.dm]: yo sorry I did see this, why is this unnecessary

Comment (AnturK) [reply code/modules/antagonists/brother/brother.dm]: You can just inline it if you only use it once.
`"Conspirators : [get_brother_names()]]`

**Pull request merged**